visidata/column.py

```diff
@@ -119,7 +120,7 @@ def __deepcopy__(self, memo):
         return self.__copy__()  # no separate deepcopy
 
     def __getstate__(self):
-        d = {k:getattr(self, k) for k in 'name width height expr keycol formatter fmtstr voffset hoffset aggstr'.split() if hasattr(self, k)}
```

Comment:
`_formatter` is the raw value

Reply:
So the idea is that there's a `formatter` property which is viewable/editable by string name on the columns sheet, but `_formatter` is the actual formatter generating function itself.  Same with `fmtstr` and `_format` (now `_formatdict`).  Then `fmtstr` would become `format` that could be either string (most cases) or dict for our enum formatter, and be viewable/editable in the same way.